tikz/__init__.py

```diff
@@ -1024,6 +1024,7 @@ def __init__(self, tempdir=None, cache=True, opt=None, **kwoptions):
         super().__init__(opt=opt, **kwoptions)
         # additional preamble entries
         self.preamble = []
+        self.document_codes = []
```

Comment:
Please insert a line comment on the line before explaining the purpose of the variable.

Also please call the variable `document_code` without the s.